spec/migration_spec.rb

```diff
@@ -26,24 +26,4 @@
       end
     end
   end
-
-  %w(foreman-proxy-content katello).each do |scenario_name|
```

Comment:
Note I dropped this test. After exploring it a bit and some thinking, this no longer seemed like it made sense to verify since we are actively wanting to migrate and change the config now as we make further changes.

Reply:
IMHO it does make sense to have a point in time to migrate from. Where this point is can be debated, but the goal is to simulate a user who installs Foreman 1.x and then upgrades to the latest scenario. We've seen in the past that old scenario + migrations can create a different scenario that we expected and has helped to catch some bugs. If anything, I'd actually like to have more of these scenarios to test our migrations. It's a lot of Ruby code we don't have other tests for.

Reply:
Is this covered by adding https://github.com/theforeman/foreman-installer/pull/437 ?

Reply:
Yes, but I'd like to merge that before dropping it here to make sure we still have decent coverage. Or keep it in for now.